docker/router/traefik.yml

```diff
@@ -0,0 +1,11 @@
+# Docker configuration backend
+providers:
+  docker:
+    defaultRule: "Host(`{{ trimPrefix `/` .Name }}.localhost`)"
```

Comment:
Just curious: are we going to stick with `.localhost` setup from now on? The usual approach of `subdomain.domain.tld` has a few advantages (mainly when using multiple subdomains and i18n-versions, plus consistency of course).

Reply:
I assume this is also a way to ditch the port-hell, right? So this is us switching to easy rememberable hostnames again, right? In that case I'd assume a regex like `localhost.<name>.<extension>` would also work, but maybe this has been a explicit decision I'm not aware of 👍

Reply:
This is the way to prevent port-hell. Because every project runs on port 80 you can't run project simultaneously without changing some configuration, but it makes is easier.

About the url naming convention: @harmenjanssen why did we choose for `.localhost` and not the `localhost.`? Wasn't it because of an idea to add SSL support?

Reply:
I think it has something to do with `.localhost` being supported by default by Traefik, but then again, since we add this line ourselves I'm not sure anymore. 

@aapit What was it again? Can you explain why `.localhost` is a better fit than `localhost.`, even though we have to configure it anyway (see above)? I know you told me already, but I can't remember. 🙏

Reply:
Mainly because `.localhost` is a pretty universal convention by which most systems will point to `127.0.0.1` with a `.localhost`, without the need to configure anything.
The rest we can pretty much handle with Traefik. I think that would be better than the current port mapping hack.
We could also choose project specific subdomains like `localhost.example.com`, which would require manually adjust `/etc/hosts` or whatever. I could live with that if it makes other stuff a lot easier.
Method 3 would be to use a universal `.grrr.io` domain, so we can dynamically point our machines to `localhost` whenever `*.grrr.io` is called upon, but it would also require a local `dnsmasq` or `bind`, something at least a bit fancier than just `/etc/hosts`.